changelog/5304.removal.1.rst

```diff
@@ -0,0 +1 @@
+``sunpy.roi.chaincode.Chaincode`` has been removed in favour of `sunpy.net.helio.Chaincode`.
```

Comment:
In general I suggest you add "the deprecated x has been removed" just so it's clear to people not paying attention we gave everyone warning.

Reply:
K